scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -152,3 +159,59 @@ def __iter__(self):
 
     def get_generated_keys(self) -> List[str]:
         return [self._name]
+
+
+class AttackPointIteratorRepeat(AttackPointIterator):
+    """
+    Attack point iterator class that iterates 
+    over a configuration a repeat amount of times.
+    """
+
+    def __init__(self,
+                 operation: str,
+                 configuration: Dict,
+                 repetitions: int = -1) -> None:
+        """Initialize the repeated iterate if repetitions is not present
```

Comment:
Missing dot somewhere maybe? The sentence seems unusually long without punctuation